getfastq.sh

```diff
@@ -102,41 +103,63 @@ function _progress_getfastq {
         printf "Bad TARGETS path '$1'.\n"
         exit 1 # Argument failure exit status: bad target path
     fi
-
-    local log_file="$(find "${target_dir}" \
-        -maxdepth 1 -type f -iname "Z_getFASTQ_*.log")"
-
-    if [[ -n "$log_file" ]]; then
-
-        # NOTE: the -- is used here to indicate the end of 'bash' command
-        #       options and the beginning of Bash script arguments.
-        # NOTE: possible non-matching greps inside 'find ... -exec bash' won't
-        #       break the script, even when 'set -e' is active.
-        printf "\n${grn}Completed:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                grep -E " saved \[| already there;" "$1"
-            ' -- {} \;
-
-        printf "\n${red}Failed:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                grep -E ".+Terminated| unable to |Not Found." "$1"
-            ' -- {} \;
-
-        printf "\n${yel}Incoming:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                dead_track=$(tail -n 3 "$1" | grep -E \
-                    " saved \[| already there;|Terminated|unable to|Not Found")
-                [[ -z $dead_track ]] && tail -n 1 "$1" && echo
-            ' -- {} \;
-        exit 0 # Success exit status
-    else
+    
+    declare -a logs=()
```

Comment:
why not symply 
```bash
logs=()
```
?

Reply:
for clarity and self-documentation... I suppose.

Reply:
Yeah, but while I was programming `log=()` did not work, so I was trying everything in my panic. I think we can go back to just `log=()`, but I guess we can keep it as is for documentation, as you say.